ichk/check.py

```diff
@@ -89,21 +89,28 @@ def compare_filesize(self):
         if isinstance(self.statinfo, Status):
             return self.statinfo
```

Comment:
This will raise TypeError when the return value is unpacked into two variables on L359.
```suggestion
             return self.statinfo, {}
```